crates/kgraph_utils/src/transformers.rs

```diff
@@ -124,6 +129,105 @@ impl IntoDirValue for api_enums::FutureUsage {
     }
 }
 
+impl IntoDirValue for api_enums::PaymentMethodType {
+    fn into_dir_value(self) -> Result<dir::DirValue, KgraphError> {
+        match self {
+            Self::Credit => Ok(dirval!(CardType = Credit)),
+            Self::Debit => Ok(dirval!(CardType = Debit)),
+            Self::Giropay => Ok(dirval!(BankRedirectType = Giropay)),
+            Self::Ideal => Ok(dirval!(BankRedirectType = Ideal)),
+            Self::Sofort => Ok(dirval!(BankRedirectType = Sofort)),
+            Self::Eps => Ok(dirval!(BankRedirectType = Eps)),
+            Self::Klarna => Ok(dirval!(PayLaterType = Klarna)),
+            Self::Affirm => Ok(dirval!(PayLaterType = Affirm)),
+            Self::AfterpayClearpay => Ok(dirval!(PayLaterType = AfterpayClearpay)),
+            Self::GooglePay => Ok(dirval!(WalletType = GooglePay)),
+            Self::ApplePay => Ok(dirval!(WalletType = ApplePay)),
+            Self::Paypal => Ok(dirval!(WalletType = Paypal)),
+            Self::CryptoCurrency => Ok(dirval!(CryptoType = CryptoCurrency)),
```

Comment:
Don't have this as an `IntoDirValue` impl. Instead have it as a private function in `kgraph_utils::mca`, since we don't want to encourage its usage.